src/Aspire.Dashboard/Components/Controls/Chart/DashpageCharts.razor

```diff
@@ -10,44 +10,27 @@
     <h3 title="@SelectedDashpage.Name">@SelectedDashpage.Name</h3>
 }
 
-<FluentGrid Spacing="3" AdaptiveRendering="true" Justify="JustifyContent.FlexStart" Class="dashpage-chart-grid">
+<section class="dashpage-chart-grid">
     @foreach (var chart in SelectedDashpage.Charts)
     {
-        if (Instruments.FirstOrDefault(i => i.Name == chart.InstrumentName) is { } instrument)
+        if (Instruments.FirstOrDefault(i => StringComparers.OtlpInstrumentName.Equals(i.Name, chart.InstrumentName)) is { } instrument)
```

Comment:
nit: string.Equals + StringComparison.OtlpInstrumentName is more idomatic in this scenario

Reply:
Seems subjective. I prefer this form as it's less code and feels more natural to me to use the right object for the operation, than passing an enum to a method to modify its behaviour. I feel like `StringComparison` only exists for historical reasons, and could be removed from the library if it weren't for backcompat requirements.

Reply:
StringComparer is for non-string scenarios, e.g. the key comparer in a dictionary:
https://learn.microsoft.com/en-us/dotnet/standard/base-types/best-practices-strings#methods-that-perform-string-comparison-indirectly

StringComparison is for string operations:
https://learn.microsoft.com/en-us/dotnet/standard/base-types/best-practices-strings#recommendations-for-string-usage

Reply:
This SO answer summerizies it well: https://stackoverflow.com/a/8918112

I feel like we've had this conversation before.